content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Announcing Kubernetes Upstream Community Group Annual Reports"
+date: 2021-06-18T10:00:00-08:00
+slug: Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports
```

Comment:
I'd write:
```suggestion
slug: kubernetes-upstream-vommunity-group-annual-reports
```
Is that better?

Reply:
```suggestion
slug: announcing-kubernetes-community-group-annual-reports
```
slight change, we are announcing them but the upstream is redundant for the official blog.

slugs should also be lower case 👍